yb-voyager/cmd/exportSchemaOptimizationReport.go

```diff
@@ -57,16 +57,16 @@ type SchemaOptimizationReport struct {
 
 	// Optimization changes applied
 	RedundantIndexChange        *RedundantIndexChange         `json:"redundant_index_change,omitempty"`
-	TableShardingRecommendation *ShardingRecommendationChange `json:"table_sharding_recommendation,omitempty"`
-	MviewShardingRecommendation *ShardingRecommendationChange `json:"mview_sharding_recommendation,omitempty"`
+	TableColocationRecommendation *ColocationRecommendationChange `json:"table_colocation_recommendation,omitempty"`
+	MviewColocationRecommendation *ColocationRecommendationChange `json:"mview_colocation_recommendation,omitempty"`
```

Comment:
do we generate the json report also for schema optimization?

Reply:
Yes, but that's only for testing purpose, it's not given to the user.

Reply:
No, it is automatically created with the HTML report but we don't give the path to the user so that way it's not shown to users